modules/wificountry.sh

```diff
@@ -15,24 +20,33 @@ function wificountry {
       ;;
 
     *)
-      if [ -e /etc/wpa_supplicant/wpa_supplicant.conf ]; then
-        if grep -q "^country=" /etc/wpa_supplicant/wpa_supplicant.conf ; then
-          sed -i --follow-symlinks "s/^country=.*/country=$country/g" /etc/wpa_supplicant/wpa_supplicant.conf
-        else
-          sed -i --follow-symlinks "1i country=$country" /etc/wpa_supplicant/wpa_supplicant.conf
-        fi
-      else
-        echo "country=$country" > /etc/wpa_supplicant/wpa_supplicant.conf
-      fi
-      iw reg set "$country" 2> "$LOGFILE";
+      for country_code in "${country_codes[@]}"
+      do
```

Comment:
@kjong think instead of case and for loop
we should just do ugly but effective case
like `"us"|"gb"|...)`